### DISCUSSION TITLE
Hyphen in vcf file cause error.

### DISCUSSION THREAD

**Discussion by Alex-Karmazin** [KarchinLab/open-cravat#252] [Usability & Feature Requests]
When vcf file contain hyphen it cause this error:
2022/04/19 14:45:00 cravat.aggregator    Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\openCravatPlugin\lib\site-packages\cravat\aggregator.py", line 155, in run
    self.cursor.execute(update_template, ins_vals)
sqlite3.OperationalError: near "-": syntax error

Line that cause error is:
##INFO=<ID=HGMD-PUBLIC_20204,Number=0,Type=Flag,Description="Variants from HGMD-PUBLIC dataset December 2020">

When I change hyphen to underscore there is no error.

**Comment by kmoad** [comment]
Sorry for the late response. A solution for this is in the queue. Likely, the solution will be to automatically convert hyphens to underscores.

**Comment by kmoad** [comment]
This is fixed in `vcf-converter` version `2.2.1`. INFO fields with a dash in the name are converted internally to an underscore. They will still show up with a dash in the GUI.

https://github.com/KarchinLab/open-cravat-modules-karchinlab/commit/b6d11dfb0ae6b030bf7cc8f0768cd33932e96b30
